cocos/animation/core/animation-handle.ts

```diff
@@ -0,0 +1,15 @@
+export interface TransformHandle {
```

Comment:
Can Handle be used as a pure number? Using an object I think would be wasteful and cache unfriendly, call xxx.destroy(handle) when you need to destroy it

Reply:
I have documented the reason